packages/js/product-editor/src/blocks/checkbox/README.md

```diff
@@ -0,0 +1,45 @@
+# woocommerce/product-checkbox-field
+
+A reusable checkbox for the product editor.
+
+_Please note that to persist a custom field in the product it also needs to be added to the WooCommerce REST API._
+
+| **attribute** | **type** | **required** | **description**                                                            |
+|--------------:|----------|--------------|----------------------------------------------------------------------------|
+| title         | string   | no           | Header that appears above the checkbox                                     |
```

Comment:
Side note: What?! That's super odd that we have this `title` attribute on a checkbox. I wonder why it is included in the component.

Reply:
I agree. Here's an example where this attribute is used:
 
<img width="350" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13437655/cf52411e-df9f-4de4-9f59-019c3920aa56">